src/client/Microsoft.Identity.Client/ApiConfig/AbstractAcquireTokenParameterBuilder.cs

```diff
@@ -334,9 +334,9 @@ public T WithB2CAuthority(string authorityUri)
             return this as T;
         }
 
-        internal /* for testing */ T WithAuthenticationScheme(IAuthenticationScheme scheme)
+        internal /* for testing */ T WithAuthenticationOperation(IAuthenticationOperation scheme)

```

Comment:
Is this internal because extensions are supposed to have InternalsVisibleTo to MSAL?